docs/UserGuide.md

```diff
@@ -128,80 +128,108 @@ Format: `view com INDEX`
 Example:
 - `view com 3`
 
-### Internship Application
+### Internship
 
-#### Adding an internship application: `add -int`
+Note: You must be on the **Companies** page in order to execute these commands.
 
-Applies for an internship. This internship will be automatically added to the list of jobs in the company
-(if company has already been created). Otherwise, it creates the company and adds this job into it.
+#### Adding an internship: `add int`
 
-Format: `add -int n/COMPANY_NAME j/JOB_TITLE [i/INDUSTRY] [p/PERIOD] [w/WAGE] [r/REQUIREMENT]...`
+Adds an internship to a company.
 
-Examples:
-- `add -int n/Google i/Software j/Software Engineer`
-- `add -int n/Google i/Software j/Software Engineer r/React w/3000 r/HTML5`
-
-**Already created the internship before?**
-
-Fret not, we have an alternative method to apply for an internship! This format adds an internship with the company
-name and job title only.
+Format: `add int INDEX j/JOB_TITLE [p/PERIOD] [w/WAGE] [r/REQUIREMENT]...`
 
-Format: `add -int n/COMPANY_NAME INDEX` - `INDEX` refers to position of internship in the company
-
-:bulb: **Tip:** `view -com INDEX` (to view a company's profile, including its list of jobs)
+Examples:
+- `add int 1 j/Software Engineer`
+- `add int 3 j/Web Developer r/React w/3000 r/HTML5`
 
-#### Deleting an internship application: `delete -int`
+#### Deleting an internship: `delete int`
 
-Deletes an internship.
+Deletes an internship from a company. The application made for the deleted internship, if any, will also be deleted.
```

Comment:
this is a really small nitpick but i feel the flow of the second sentence can be improved? just feel a bit weird having the if any in the middle.
maybe something like any applications with the deleted internship, will also be deleted?

Reply:
Right now we are only allowing one application for an internship. So perhaps your suggestion may be quite misleading too.
I have updated to:
The application (if any) made with this internship will also be deleted.

Hopefully its clearer this way!